snapshots/devmapper/README.md

```diff
@@ -26,6 +26,7 @@ The following configuration flags are supported:
   should be the same as in `/dev/mapper/` directory
 * `base_image_size` - defines how much space to allocate when creating the base device
 * `async_remove` - flag to async remove device using snapshot GC's cleanup callback
+* `discard_blocks` - whether to discard blocks when removing a device. This is especially useful for returning disk space to the filesystem when using loopback devices.
```

Comment:
What are the drawbacks of the option? If it doesn't have much drawbacks, can't we do that by default?

Reply:
I'm not sure if blkdiscard is always a cheap operation. I don't think that's an issue for dev setups where we're most likely to see loop files, but maybe there are prod setups where it could introduce latency.